### PR TITLE
[PW-6717] Refactor e2e tests for Playwright

### PR DESCRIPTION
**Pull request by cenkiravul**
## Summary

**Fixed issue**:  PW-6717

### CODE REVIEW

**Comment by maassenbas** [comment]
@cenkiravul , let's change this PR into `develop` instead of `main`

File: tests/playwright/sfcc.config.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    },
+
+    {
+      name: 'edge',
+      use: {
+        channel: 'msedge',
+      },
+    },
+
+    /* Test against mobile viewports. */
```
Comment (wboereboom) [inline tests/playwright/sfcc.config.js:90]: do we need to keep this commented code?

File: tests/playwright/package.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+  "devDependencies": {
+    "@playwright/test": "^1.21.0"
+  },
+  "name": "adyen_salesforce_e2e_tests",
+  "version": "1.0.0",
+  "description": "e2e tests for Adyen SalesForce Cartridges",
+  "main": "index.js",
+  "dependencies": {
```
Comment (wboereboom) [inline tests/playwright/package.json:9]: should these be devDependencies?

**Review by wboereboom — CHANGES_REQUESTED** [review]
Looks good. 2 small remarks :)

File: tests/playwright/sfcc.config.js (hunk 1 above)
Comment (cenkiravul) [reply tests/playwright/sfcc.config.js:90]: It's a default config file, would be nice to keep them imho if we ever need to enable test cases for mobile viewports.

File: tests/playwright/package.json (hunk 2 above)
Comment (cenkiravul) [reply tests/playwright/package.json:9]: We only install playwright as dev dependency, the rest is added by playwright during installation. I don't have a strong opinion on this.

**Review by wboereboom — APPROVED** [review]

**Review by zaiddreakh — APPROVED** [review]

**Pull request merged**